shim/CustomEvent.js

```diff
@@ -1,9 +1,8 @@
 (function (window) {
   'use strict';
-
-  /**
-   * Detect full support
-   */
+  //
```

Comment:
Oh, no, I had it all commented out so changed the comment style to be wrappable.

I'll change it back.